src/Sylius/Bundle/ApiBundle/Resources/config/services/filters.xml

```diff
@@ -26,6 +26,20 @@
             <tag name="api_platform.filter" />
         </service>
 
+        <service id="sylius.api.archived_promotions_filter" parent="api_platform.doctrine.orm.exists_filter" public="true">
```

Comment:
I see that file has multiple naming schemes, I personally like `sylius.api.catalog_promotion_enabled_filter` scheme better.

```suggestion
        <service id="sylius.api.promotion_archived_filter" parent="api_platform.doctrine.orm.exists_filter" public="true">
```